src/components/instances_page.rs

```diff
@@ -90,8 +119,9 @@ impl SimpleComponent for InstancesPage {
     fn update(&mut self, msg: Self::Input, sender: ComponentSender<Self>) {
         match msg {
             InstancesPageInput::FetchInstances => {
+                let filter =  self.instances_search_buffer.text().as_str().to_owned().clone();
```

Comment:
Please lowercase the string, so that users still get results when using caps lock.